crates/transaction-pool/src/lib.rs

```diff
@@ -325,6 +325,20 @@ where
     fn on_propagated(&self, txs: PropagatedTransactions) {
         self.inner().on_propagated(txs)
     }
+
+    fn get_all_transactions_by_sender(
```

Comment:
we need the same thing in the other layers as well, see `fn pooled_transactions` for example.

Reply:
will add ser!